ai/STA/Strategy/strategy_book.py

```diff
@@ -62,10 +62,20 @@ def __init__(self):
                               'PreparePenaltyOffense': PreparePenaltyOffense,
                               'OffenseKickOff': OffenseKickOff,
                               'DefenseWallNoKick': DefenseWallNoKick,
-                              'pStop': DoNothing
+                              'DoNothing': DoNothing
                               }
+        self.default_strategies = ['Offense',
+                                   'DefenseWall']
 
-    def get_strategies_name_list(self) -> List[str]:
+        for name, strategy_class in self.strategy_book.items():
+            assert name == strategy_class.__name__, \
```

Comment:
python assertion should not be use for data validation. You should raise an KeyError exception. Same for deafult strategy.